### PR TITLE
Add helloAhmed

### PR DESCRIPTION
**Pull request by Hossam-Tarek**
Ahmed please admit the pull request

### CODE REVIEW

File: helloAhmed [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Hello Ahmed
```
Comment (AhmedFawzyMetwaly) [inline helloAhmed:1]: You forgot to end statement with semicolon.

Comment (Hossam-Tarek) [reply helloAhmed:1]: oh my god i forgot to add it 
i will do it quickly

Comment (AhmedFawzyMetwaly) [reply helloAhmed:1]: No problem.
Do it as soon as possible.

File: helloAhmed [hunk 2]
```diff
@@ -1,2 +1,3 @@
-Hello Ahmed
-How are you 
+Hello Ahmed;
```
Comment (AhmedFawzyMetwaly) [inline helloAhmed:1]: All is well.

**Pull request merged**